src/main/java/com/rackspace/salus/event/manage/services/TasksService.java

```diff
@@ -202,4 +150,114 @@ public void deleteAllTasksForTenant(String tenant) {
     eventEngineTaskRepository.findByTenantId(tenant, Pageable.unpaged())
         .forEach(task -> deleteTask(tenant, task.getId()));
   }
+
+  @Transactional
+  public EventEngineTask updateTask(@Valid EventEngineTask update) {
+    EventEngineTask eventEngineTask = eventEngineTaskRepository.findByTenantIdAndId(update.getTenantId(), update.getId()).orElseThrow(() ->
+        new NotFoundException(String.format("No Event found for %s on tenant %s",
+            update.getId(), update.getTenantId())));
+    log.info("Updating event engine task={} with new values={}", update.getId(), update);
+    boolean needsUpdate = false;
+    if(!StringUtils.isEmpty(update.getName()) && !eventEngineTask.getName().equals(update.getName()))  {
+      log.info("changing name={} to {} ",eventEngineTask.getName(), update.getName());
```

Comment:
In the spirit of the logging conversations we've had, using this may help us once we have a better overall kibana solution.  There's more places below where we can do similar

```suggestion
      log.info("changing name={} to updatedName={} ",eventEngineTask.getName(), update.getName());
```

I like that you're adding these logs; we don't have enough and I think `info` is a valid level for these updates.